SECURITY.md

```diff
@@ -55,10 +55,7 @@ Security.insertProviderAt(Conscrypt.newProvider(), 1);
 JDK versions prior to Java 9 do not support ALPN and are either missing AES GCM
 support or have 2% the performance of OpenSSL.
 
-We recommend most users use grpc-netty-shaded, which includes netty-tcnative on
-BoringSSL. It includes pre-built libraries for 64 bit Windows, OS X, and 64 bit
-Linux. For 32 bit Windows, Conscrypt is an option. For all other platforms, Java
-9+ is required.
+We recommend most users use [grpc-netty-shaded](#tls-with-grpc-netty-shaded).
```

Comment:
How about rewording this to say something like "users of xDS management protocol" which is what the original issue was?

Reply:
Waiting to address the comment so we can go ahead and merge this....